pycqed/instrument_drivers/physical_instruments/ZurichInstruments/ZI_HDAWG8.py

```diff
@@ -749,5 +749,8 @@ def calibrate_CC_dio_protocol(self, CC, verbose=False, repetitions=1):
 
         # And configure the delays
         self.setd('raw/dios/0/delays/*', min_valid_delay)
-        # If succesful return True
+
+        # If succesful clear all errors and return True
+        self.clear_errors()
```

Comment:
It would be better to ensure that only clear errors related to DIO calibration are cleared after calibration, as this may lead a user to overlook any other potential problems after calibration. 
Nonetheless, I believe this is not a blocking problem for now.